indico/modules/rb/client/js/common/rooms/RoomEditModal.module.scss

```diff
@@ -13,11 +13,15 @@
 }
 
 .submit-message {
-    margin-bottom: 10px;
+    margin: 0 0 1em !important;
 }
 
 .room-form {
     :global(.fields) > :global(.field) {
         flex-grow: 1 !important;
     }
 }
+
+:global(.DateRangePicker_picker) { // sass-lint:disable-line class-name-format
+    z-index: 10;
+}
```

Comment:
Yes, should be enough. Agree. 
Not sure if there is any implication in applying this everywhere though.

Reply:
Should be safe. Just make sure it's before the 1001 z-index for the fullscreen one in that file. 

FWIW I suggested @Leats to do the exact same in her task since she also had a case of a switch appearing above a date picker, so if this solution works for you as well I'm actually thinking about just pushing it straight to master since both of you need it anyway.

Reply:
I cannot confirm it works for me since react-dates' `_datepicker.css` is now overriding the new `dates.scss` style.

![image](https://user-images.githubusercontent.com/12183954/79979954-42df9380-849a-11ea-98bf-8b27b0ed84dd.png)

We can solve it by adding a !important though.

Reply:
ah I guess we're including their css later... `!important` seems reasonable then.